docs/openapi/components/schemas/common/AdditionalProductCodeRegistrationCredential.yml

```diff
@@ -7,14 +7,11 @@ description: Ecommerce Additional Product Code Registration Credential
 type: object
 properties:
   type:
-    oneOf:
-      - type: array
-        items:
-          type: string
-          enum:
-            - AdditionalProductCodeRegistrationCredential
-      - type: string
-        const: AdditionalProductCodeRegistrationCredential
+    type: array
```

Comment:
Double typed here?

Reply:
@mprorock, yes, the type element is an array type.